schemachange/schemachange.h

```
@@ -263,6 +264,7 @@ enum schema_change_rc {
     SC_ABORTED,
     SC_PREEMPTED,
     SC_DETACHED,
+    SC_FUNC_IN_USE,
```

Can we return a proper error from replica corresponding to this error code?